README.md

```diff
@@ -53,3 +53,42 @@ Special thanks go to:
 - Github for streamlining project organization
 - Everyone on sketchfab for inspiration and actual models!
 - You!
+
+
+## User Guide
+Thank you for your interest in our project. To be able to try it out there are a few things you need to install and set up.
+
+First of all you need to install the SMCE from [SMCE](https://github.com/ItJustWorksTM/smce-gd) 
+Follow the instructions on the github page. 
+
+## Set up
+If you installed the SMCE correctly you will be within the godot folder. Godot\app_userdata\SMCE\mods here you will paste in the mods from the project.
+
+From the root folder go to group-02\godot-resources\mods in these folders you will find pck files. Copy them and paste them within the mods folder in SMCE 
```

Comment:
We should add a step for cloning our code - to refer to here.

Reply:
I could add this no problemos. But maybe add a link to git hubs description of cloning a directory?

Reply:
included updates in new commit